scripts/build/windows.bat

```diff
@@ -226,6 +233,10 @@ signtool verify /pa /v %package_output%\Etcher.exe
 
 upx -9 %package_output%\*.dll
 
+cd %package_output%^
+ && 7z a -tzip ..\installers\%package_name%.zip *^
```

Comment:
We can always include `7z.exe` on this repo and get the best out of both worlds. Thoughts?